static/app/components/codecov/container/container.tsx

```diff
@@ -0,0 +1,104 @@
+import {useState} from 'react';
+
+import {CodecovContext} from 'sentry/components/codecov/context/context';
+import {useLocalStorageState} from 'sentry/utils/useLocalStorageState';
+import {useLocation} from 'sentry/utils/useLocation';
+import {useNavigate} from 'sentry/utils/useNavigate';
+import useOrganization from 'sentry/utils/useOrganization';
+
+// Types
+export type CodecovContextTypes = {
```

Comment:
```suggestion
export type CodecovContext = {
```
There is no need to annotate this any further